e2e/tests/localAuthorityMaintainsCaseAfterSubmission_test.js

```diff
@@ -63,3 +63,35 @@ Scenario('local authority provides a statements of service', async (I, caseViewP
   I.seeAnswerInTab(7, 'Recipients 2', 'How were they sent?', recipients[1].sentBy);
   I.seeAnswerInTab(8, 'Recipients 2', 'Recipient\'s email address', recipients[1].email);
 });
+
+Scenario('local authority upload placement application', async (I, caseViewPage, placementEventPage) => {
```

Comment:
maybe we should also test scenario with only 1 children, as it changes which pages are displayed; currently we only cover scenario with 2 children in our test - but if this story is urgent then I wouldn't worry for now

Reply:
i did it on purpose like that, of course we can have another test but it will required new case to be created and submitted as we should not remove children after submission (possible in system but invalid from business pov). My concern was about performance of such solutions as our e2e test are already slow. Most common scenario is with 1 child but due to CCD bug for which we have workaround (only for multi children cases) i wonted to have this scenario covered as there is risk that functionality will broke when CCD fix their issue